packages/cozy-client/src/models/note_schema.js

```diff
@@ -0,0 +1,526 @@
+// taken from a debug of @atlakit/editor/editor-core/create-editor/create-editor
+// L139 (new Schema({nodes ,marks}))
+// static because the @atlaskit code base requires a real navigator
+// TODO: either find and exclude plugins requiring interaction
+//       or running a JSDOM faking a navigator
+
+export const nodes = [
```

Comment:
Actually I'm not sure this is a `model`? In my mind they were collections of functions related to a doctype, this file isn't really that.
Then again I'm not sure what better place there would be…